### ISSUE TITLE
Screen rotation / landscape mode shows, but immediately switches back to portrait

### ISSUE THREAD

**Issue by hrehfeld** [corona-warn-app/cwa-wishlist#86]
## Describe the bug
Screen rotation / landscape mode shows for `<` 1 s, but immediately switches back to portrait.

## Expected behaviour

Show app in landscape mode.

## Steps to reproduce the issue

- Turn on screen rotation in system settings.
- Turn phone so it shows desktop (seperate setting) or any other app in landscape mode.
- Open CWA.

- [ Screen shows in landscape, then immediately switches back to portrait. ]

## Technical details

- Android Lineage OS 16

## Additional context

People using their old phones with broken screens is the main reason this is important. Especially if your main phone is google free... :crying_cat_face: 

-------------------
Internal Tracking Id: EXPOSUREAPP-2109

**Comment by Blackjacx** [comment]
Why do we need to bother with landscape mode for a **fire-and-forget app** (name coined by an owner in another issue)?! Especially for phones and such a simple app this is not really needed.

**Comment by Ein-Tim** [comment]
@Blackjacx 
Just a small note from me, since it looks like you just joined the CWA Community:
Not all Issues here in the Wishlist are currently being worked on. There are here for "Long Term visibility", but if you are watching these Issue they are often just handed over to the next and next Asignee but they never really get implemented.
Or better let's say: 
That Issues from the Wishlist get implemented is more the exception than the rule.

**Comment by hrehfeld** [comment]
> People using their old phones with broken screens is the main reason this is important. Especially if your main phone is google free...

E.g. the new screens added in one of the previous updates are not touchable on my spare phone's broken screen.

**Comment by stherold** [comment]
Oh interesting reason 😅 Never though as app developer that screen rotation makes sense because of this. Thx for the hint 👍

**Comment by mvglasow** [comment]
Here are some additional thoughts I mentioned to the maintainers of the CCTG fork (who suggested I report this upstream):

180° rotation could be achieved quite easily and would be useful in use cases where the device is on a charger and I would like to move the cable to a position where it does not get in my way.

+/- 90° rotation is trickier as the UX might be less than satisfactory without additional UI changes.

For the **main screen**, which is essentially a tile-based layout, this could be addressed through a two-abreast layout (note that the suggestion refers to CCTG, CWA might look slightly different):

```
 ——————————————————————————————————————————————————
| Risk                     | Have you been tested? |
 ——————————————————————————————————————————————————
| Statistics (multiple tiles, scroll horizontally) |
 ——————————————————————————————————————————————————
| Contact Journal          | FAQ                   |
 ——————————————————————————————————————————————————
```

Neigboring tiles are already of similar height—the odd one out is the Donate tile, which is far taller than the others, and presumably the only one not present in upstream.

For the **Overview** activity, the graphic could go on one side of the screen, with the explanations in a column next to it. Definitions could be shown either underneath or in a third column.

**App Information** and **Settings** are similar to any Preferences activity on Android: at most, some padding on the sides may be required.

**Exposure Notification** and **Prioritized Background Activity** can be solved with a multi-column layout similar to Overview.

**Notifications** is essentially a Preferences screen with a graphic on top; the image and settings could be arranged in two columns.

While it does take some work to implement, it should be fairly straightforward and “mechanical”, one-off as long as the screen layouts do not change dramatically.

**Comment by MikeMcC399** [comment]
@mvglasow 
> 180° rotation could be achieved quite easily and would be useful in use cases where the device is on a charger and I would like to move the cable to a position where it does not get in my way

I picked up this point and submitted a specific Android enhancement request https://github.com/corona-warn-app/cwa-app-android/issues/2274. I would also like to be able to have a charger connected and stand the device upright.

**Comment by d4rken** [comment]
This bug was fixed by #738
